server/api/contest_handler.py

```diff
@@ -178,3 +177,25 @@ def get_submitted_to_contests(user_id):
             counter += 1
         return json.dumps(dictionary, default=str)
 
+@contest_handler.route('/contest_winner', methods=['POST'])
+def set_contest_winner():
+    # Does contest exist?
+    request_json = request.get_json()
+
+    contest_id = request_json.get("contest_id")
+    winning_submission_id = request_json.get("winning_submission_id")
```

Comment:
a better param name would probably be `winning_user_id` since it looks like it's a user id